onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6652,11 +6652,13 @@ def aten_positive(self: TensorType) -> TensorType:
         "aten::pow.Tensor_Scalar",
         "_operator::pow",
     ),
-    traceable=True,
+    trace_only=True,
 )
 def aten_pow(self: TReal, exponent: TTensor) -> TReal:
     """pow(Tensor self, Tensor exponent) -> Tensor"""
 
+    if isinstance(self, (int, float)) and hasattr(exponent, "dtype"):
```

Comment:
Instead, I would split the overload "aten::pow.Scalar" out into a separate function to handle this case only. We should actually separate out all of the different cases because they are different.

Reply:
done